pkg/services/provisioning/dashboards/validator_test.go

```diff
@@ -93,11 +92,10 @@ func TestDuplicatesValidator(t *testing.T) {
 		fakeStore := &fakeDashboardStore{}
 		r, err := NewDashboardFileReader(cfg, logger, nil, fakeStore)
 		require.NoError(t, err)
-		folderID, _, err := r.getOrCreateFolder(context.Background(), cfg, fakeService, folderName)
+		_, folderUID, err := r.getOrCreateFolder(context.Background(), cfg, fakeService, folderName)
```

Comment:
is the folderID used somewhere in the getOrCreateFolder function? Maybe that could be removed